src/main/java/uk/gov/di/ipv/cri/passport/service/ConfigurationService.java

```diff
@@ -167,4 +172,11 @@ public long getBearerAccessTokenTtl() {
                 .map(Long::valueOf)
                 .orElse(DEFAULT_BEARER_TOKEN_TTL_IN_SECS);
     }
+
+    public URI getDynamoDbUri() {
+        URI defaultDynamoDbUri = URI.create(DEFAULT_DYNAMODB_URI);
+        return Optional.ofNullable(System.getenv("DYNAMODB_URI"))
```

Comment:
I'm a bit unsure about the changes to the DynamoDB DataStore class and this. Where will this env var come from now? I could be missing something but I think its just going to connect to local dynamo each time now unless we have a DYNAMODB_URI set?